include/sgx/enclaveConfig.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+/* Main hardcoded memory sizes */
+#define ONE_KB_BYTES 1024
+#define ONE_MB_BYTES (1024 * 1024)
+#define FAASM_SGX_WAMR_HEAP_SIZE (32 * ONE_MB_BYTES)
+#define FAASM_SGX_WAMR_MODULE_ERROR_BUFFER_SIZE 128
+#define FAASM_SGX_WAMR_INSTANCE_DEFAULT_HEAP_SIZE (8 * ONE_KB_BYTES)
+#define FAASM_SGX_WAMR_INSTANCE_DEFAULT_STACK_SIZE (8 * ONE_KB_BYTES)
+
+/* Thread Control Structure (TCS)
```

Comment:
This comment seems a bit long for a single `#define` and includes some general info about SGX, perhaps a shorter comment saying specifically what this parameter does in the context of our application, then a link to the TCS docs would be better.